src/frontends/common/include/openvino/frontend/utils/random_normal.hpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/reshape.hpp"
+#include "ngraph/output_vector.hpp"
+#include "openvino/frontend/visibility.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/opsets/opset12.hpp"
+#include "openvino/pass/graph_rewrite.hpp"
+#include "transformations/rt_info/disable_fp16_compression.hpp"
+#define _USE_MATH_DEFINES
+#include <math.h>
+
+namespace ov {
+namespace frontend {
+
+/// \brief Creates a random normal tensor with the given shape and type.
+/// \details Uses Box-Mueller algorithm to generate random numbers from a Gauassian distribution
+/// \param sizes Shape of the output tensor
+/// \param target_type Type of the output tensor
+/// \param mean Mean of the distribution
+/// \param scale Standard deviation of the distribution
+/// \param seed Seed for the random number generator
+inline std::pair<OutputVector, NodeVector> make_random_normal(const Output<Node>& sizes,
```

Comment:
nit: I think better would be to pass NodeRegistry inside as reference. Maybe it also makes sense to have another overloaded function without NodeRegistry, which will call function with NodeRegistry internally.